api/backtest_async.resolver.go

```
@@ -0,0 +1,198 @@
+		Status:        domain.BacktestJobStatusPending,
+		CurrentStage:  domain.StageInitializing,
+		ProgressPct:   0,
+	}
+	createdJob, err := h.BacktestJobRepository.Create(backtestJob)
+	if err != nil {
+		returnErrorJson(fmt.Errorf("failed to create backtest job: %w", err), c)
+		return
+	}
+
+	// Update stage to running
+	h.BacktestJobRepository.UpdateStage(createdJob.BacktestJobID, domain.StageLoadingPriceData, 5)
```

golangci-lint (2.11.4)

[error] Error return value of `h.BacktestJobRepository.UpdateStage` is not checked

(errcheck)